libs/react-design-system/src/immersionFacile/components/rs-autocomplete/RSAutocomplete.tsx

```diff
@@ -0,0 +1,95 @@
+import { fr } from "@codegouvfr/react-dsfr";
+import { InputProps } from "@codegouvfr/react-dsfr/Input";
+import React from "react";
+import Select, { Props as SelectProps, GroupBase } from "react-select";
+import { useStyles } from "tss-react/dsfr";
+import Styles from "./RSAutocomplete.styles";
+
+export type OptionType<T> = { value: T; label: string };
+
+export type RSAutocompleteProps<T> = InputProps.Common &
+  InputProps.RegularInput & {
+    selectProps?: SelectProps<
+      OptionType<T>,
+      false,
+      GroupBase<OptionType<T>>
+    > & {
+      isDebouncing?: boolean;
+    };
+    initialInputValue?: string;
+  };
+
+type Capitalize<T extends string> = T extends `${infer First}${infer Rest}`
+  ? `${Uppercase<First>}${Rest}`
+  : T;
+
+export type RSAutocompleteComponentProps<
+  K extends string,
+  T,
+> = RSAutocompleteProps<T> &
+  Record<`on${Capitalize<K>}Selected`, (value: T) => void> &
+  Record<`on${Capitalize<K>}Clear`, () => void>;
+
+const prefix = "im-select";
+
+export const RSAutocomplete = <T,>({
+  state,
+  stateRelatedMessage,
+  label,
+  hintText,
+  className,
+  selectProps,
+}: RSAutocompleteProps<T>) => {
+  const hasError = state === "error";
+  const { cx } = useStyles();
+  return (
+    <div
+      className={cx(fr.cx("fr-input-group"), className)}
+      id={selectProps?.id}
+    >
+      <label className="fr-label" htmlFor={selectProps?.inputId}>
```

Comment:
utiliser `fr.cx` dans le composant (ici et plus bas)

```suggestion
      <label className={fr.cx("fr-label")} htmlFor={selectProps?.inputId}>
```